### PR TITLE
add init.d script and etc

### PR DESCRIPTION
**Pull request by lego12239**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	unxz -c zme_radiotools.txz | ( cd $(ZME_RTHOME); tar -x --exclude="zme_radiotools.crontab.in" --exclude="zme_radiotools.initd.in" --no-same-owner )
+	sed -re 's/@ZME_RTHOME@/$(subst /,\/,$(ZME_RTHOME))/' zme_radiotools.crontab.in > /etc/cron.d/zme_radiotools
+	sed -re 's/@ZME_RTHOME@/$(subst /,\/,$(ZME_RTHOME))/' zme_radiotools.initd.in > /etc/init.d/zme_radiotools
+	chmod a+rx /etc/init.d/zme_radiotools
+	@echo
+	@echo
+	@echo '##########################################'
+	@echo '#                  INFO                  #'
+	@echo '##########################################'
+	@echo Now you can add /etc/init.d/zme_radiotools to autostart:
```
Comment (PoltoS) [inline Makefile:25]: Let's add install-autostart command and do in by default in all:

```all: install install-autostart```

File: zme_dummy.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
```
Comment (PoltoS) [inline zme_dummy.py:1]: What is this file?

File: zme_radiotools.crontab.in [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#MAILTO=""
```
Comment (PoltoS) [inline zme_radiotools.crontab.in:1]: Why not logrotate?

**Review by PoltoS — CHANGES_REQUESTED** [review]

File: zme_dummy.py (hunk 2 above)
Comment (lego12239) [reply zme_dummy.py:1]: This file is just for testing of init.d script.

Comment (PoltoS) [reply zme_dummy.py:1]: Do we need it permanently here?

File: Makefile (hunk 1 above)
Comment (lego12239) [reply Makefile:25]: ok

File: zme_dummy.py (hunk 2 above)
Comment (lego12239) [reply zme_dummy.py:1]: It's complicated question :-). It's better to save it here near other related scripts, than keep it separately.

File: zme_radiotools.crontab.in (hunk 3 above)
Comment (lego12239) [reply zme_radiotools.crontab.in:1]: logrotate can't work without rotation, but we need not rotation, we just need to remove old files.

Comment (lego12239) [reply zme_radiotools.crontab.in:1]: Also, support for SIGHUP (or something like this) is needed in scripts to logrotate work properly.

**Review by p0lyg0n1 — APPROVED** [review]

**Review by p0lyg0n1 — APPROVED** [review]

**Review by p0lyg0n1 — APPROVED** [review]

**Review by p0lyg0n1 — APPROVED** [review]

File: zme_radiotools.crontab.in (hunk 3 above)
Comment (PoltoS) [reply zme_radiotools.crontab.in:1]: ok

File: zme_dummy.py (hunk 2 above)
Comment (PoltoS) [reply zme_dummy.py:1]: ok

**Pull request merged**